mailpoet/lib/Automation/Engine/Storage/WorkflowStorage.php

```diff
@@ -131,6 +131,13 @@ public function getActiveWorkflowsByTrigger(Trigger $trigger): array {
     }, (array)$data);
   }
 
+  public function flush(): bool {
```

Comment:
Wouldn't `truncate` be a better method name? This can be confused with Doctrine's `flush`, which does something else (and much safer 😄) — it syncs changes from memory to a DB. I guess this could also be somewhere in the tests only but no strong opinion about where it should lie.

Reply:
Good catch! Yes, definitely.

Reply:
Addressed in afd1e0cf1